docs/getting-started/next-up.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Next up
+description: Start your IPFS journey here! Learn how the Interplanitery File System works, install commonly used tools, and get to grips with basic d-web concepts.
```

Comment:
Oh wow, thanks for finding this info @AlexxNica!

> We can also create an issue to discuss this.

I don't really want to open this up in an issue for fear of bike-shedding, but we can do if folks think a discussion would be beneficial. 

I reckon we should change any instances of _DWeb_, _dweb_, and _dWeb_ instances to _d-web_, since these read like the word _dweeb_. Separating the _d_ from the _web_ encourages readers to internally pronounce the two syllables separately.